cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -32,19 +33,21 @@ import (
 
 // Reporter reports data to collector over gRPC.
 type Reporter struct {
-	collector api_v2.CollectorServiceClient
-	agentTags []model.KeyValue
-	logger    *zap.Logger
-	sanitizer zipkin2.Sanitizer
+	collector                  api_v2.CollectorServiceClient
+	agentTags                  []model.KeyValue
+	logger                     *zap.Logger
+	filterProcessTagsSlice []string
```

Comment:
the Reporter should not care what kind of clean-up is performed on the spans. It can accept a span sanitizer, similar to zipkin sanitizer.